core/frontend/src/types/system-information/info.ts

```diff
@@ -0,0 +1,15 @@
+/** Base structure that provides information about OS */
```

Comment:
Are Info and KernelMessage the same?

Reply:
No, `Info` is OS specific, `KernelMessage` is for each kernel message. 
As in the documentation:
```
/** Base structure that provides information about kernel message */
```

Reply:
But are they any different? They seem to have the same parameters.

Reply:
Oh, this is a development file. I'll remove.